roles/nfd_operator_deploy_custom_commit/tasks/build.yml

```diff
@@ -0,0 +1,129 @@
+---
+- name: Find a node with our NTO/TuneD fuse label
+  command: oc get nodes -l "{{ tuned_module_fuse_label }}" -oname
+  register: fuse_nodes
+
+- name: Find a worker node to label for loading the fuse module with NTO/TuneD
+  command: oc get nodes -l node-role.kubernetes.io/worker -oname
+  when: not fuse_nodes.stdout
+  register: worker_nodes
+
+- name: Mark the node with our NTO/TuneD fuse label
+  command: oc label "{{ worker_nodes.stdout_lines[0] }}" "{{ tuned_module_fuse_label }}="
+  when: not fuse_nodes.stdout
+
+- name: Apply NTO/TuneD fuse module profile manifest
+  command: oc apply -f "{{ tuned_module_fuse }}"
+
+- name: Apply the CI utils manifest
+  command: oc apply -f "{{ nfd_operator_ci_utils }}"
+
+- name: Authorize 'ns/nfd-operator-ci' Pods to access ns/nfd-operator-ci-utils images
+  command:
+    oc policy add-role-to-user
+              system:image-puller system:serviceaccount:nfd-operator-ci:default
+              --namespace=nfd-operator-ci-utils
+
+- name: Apply the imagestream manifest
+  command: oc apply -f "{{ nfd_operator_imagestream }}"
+
+- name: Apply the image helper builder manifest
+  command: oc apply -f "{{ nfd_operator_helper_image_builder }}"
+
+- block:
+  - name: Wait for the helper image to be built
+    command:
+      set -o pipefail;
+      oc get pod -lopenshift.io/build.name
+                 --field-selector=status.phase=Succeeded
+                 --no-headers
+                 -oname
+                 -n nfd-operator-ci-utils
+    register: wait_helper_image
+    until:
+      - wait_helper_image.rc != 1
+      - wait_helper_image.stdout != ""
```

Comment:
...?

Reply:
the `command` specified in the task is:
```
              set -o pipefail;
              oc get pod -lopenshift.io/build.name
                         --field-selector=status.phase=Succeeded
                         --no-headers
                         -oname
```
This is a valid `shell` task, but not a valid `command`. I'm wondering why it doesn't simply fail with `set: command not found` instead of the tricky error above, but that's not relevant for the PR ;-)

Reply:
ah ok

Reply:
@ArangoGutierrez this one isn't fixed, you need to remove `set -o pipefail;`